cmd/plugin-backend.go

```diff
@@ -11,17 +11,18 @@ import (
 )
 
 var (
-	portArg             = flag.Int("port", 0, "server port to listen on (default: 9443)\nports 9444 and 9445 reserved for other use")
-	certArg             = flag.String("cert", "", "cert file path to enable TLS (disabled by default)")
-	keyArg              = flag.String("key", "", "private key file path to enable TLS (disabled by default)")
-	featuresArg         = flag.String("features", "", "enabled features, comma separated.\noptions: ['acm-alerting', 'incidents', 'dev-config']")
-	staticPathArg       = flag.String("static-path", "", "static files path to serve frontend (default: './web/dist')")
-	configPathArg       = flag.String("config-path", "", "config files path (default: './config')")
-	pluginConfigArg     = flag.String("plugin-config-path", "", "plugin yaml configuration")
-	logLevelArg         = flag.String("log-level", logrus.InfoLevel.String(), "verbosity of logs\noptions: ['panic', 'fatal', 'error', 'warn', 'info', 'debug', 'trace']\n'trace' level will log all incoming requests\n(default 'error')")
-	alertmanagerUrlArg  = flag.String("alertmanager", "", "alertmanager url to proxy to for acm mode")
-	thanosQuerierUrlArg = flag.String("thanos-querier", "", "thanos querier url to proxy to for acm mode")
-	log                 = logrus.WithField("module", "main")
+	portArg                = flag.Int("port", 0, "server port to listen on (default: 9443)\nports 9444 and 9445 reserved for other use")
+	certArg                = flag.String("cert", "", "cert file path to enable TLS (disabled by default)")
+	keyArg                 = flag.String("key", "", "private key file path to enable TLS (disabled by default)")
+	featuresArg            = flag.String("features", "", "enabled features, comma separated.\noptions: ['acm-alerting', 'incidents', 'dev-config']")
+	staticPathArg          = flag.String("static-path", "", "static files path to serve frontend (default: './web/dist')")
+	configPathArg          = flag.String("config-path", "", "config files path (default: './config')")
+	pluginConfigArg        = flag.String("plugin-config-path", "", "plugin yaml configuration")
+	logLevelArg            = flag.String("log-level", logrus.InfoLevel.String(), "verbosity of logs\noptions: ['panic', 'fatal', 'error', 'warn', 'info', 'debug', 'trace']\n'trace' level will log all incoming requests\n(default 'error')")
+	alertmanagerUrlArg     = flag.String("alertmanager", "", "alertmanager url to proxy to for acm mode")
+	thanosQuerierUrlArg    = flag.String("thanos-querier", "", "thanos querier url to proxy to for acm mode")
+	log                    = logrus.WithField("module", "main")
+	persesDashboardsUrlArg = flag.String("perses-dashboards", "", "perses API endpoint to fetch dashboards")
```

Comment:
I don't think we need this url here as is known from the CR plugin side proxy.

Reply:
I removed this in the latest push.